plenoptic/simulate/canonical_computations/steerable_pyramid_freq.py

```diff
@@ -93,16 +94,14 @@ def __init__(self, image_shape, height='auto', order=3, twidth=1, is_complex=Fal
         self.downsample = downsample
         self.tight_frame = tight_frame
```

Comment:
self.tight_frame = True if norm="ortho" else False
Alternatively we could just do away with tight_frame altogether and just deal w/ conditioning on self.fft_norm.

Reply:
yeah only thing is I dont want people to assume that setting "ortho" is the only thing that makes it tight frame but idk maybe that's not a big deal since we hide it from the user anyway.